app/js/beacons/beaconController.js

```diff
@@ -0,0 +1,15 @@
+/*
+ * beaconController
+ * Main beacon view control.
+ */
+function beaconController($scope, beaconModel, beaconService) {
+    'use strict';
+
```

Comment:
For completeness, I've been declaring all my scope variables here, even if it's just for user input that's later passed to function. So in this case, something like `$scope.beacons = {users: []};`